src/types/union.test.ts

```diff
@@ -112,12 +113,12 @@ testTypeImpl({
     ],
 });
 
-const ObjectUnion = union([object({ tag: literal('a'), a: string }), object({ tag: literal('b'), b: number.autoCast })]);
+const ObjectUnion = union([object({ tag: literal('a'), a: string }), object({ tag: keyof({ b: null, c: null }), b: number.autoCast })]);
 
 test('ObjectUnion examples', () => {
     expect(createExample(ObjectUnion, 1)).toMatchInlineSnapshot(`
     Object {
-      "b": 0.02,
+      "b": 0.03,
```

Comment:
Ok, itt `literalValue` hoogt `keyof` de `seed` van de `CreateExampleVisitor` wel op, waardoor dit nu hoger uit komt.. 🤔  Deze change was niet obvious. 😂

Reply:
Die examples zijn ook een beetje gek, die zijn meer als inspiration. We zouden ooit de OpenAPI generator als lib kunnen toevoegen. 😄